content/themes-for-peter.json

```diff
@@ -1,17 +1,17 @@
 {
     "title":"Setting up the PIM",
-    "description":"Having trouble with your PIM catalog structure? Don't know how to set up permissions? Don't worry. Here, you are in good care.",
+    "description":"Having trouble with your PIM catalog structure? Don't know how to set up permissions? Not to worry, you have come to the right place!",
     "img":"illus-setting-up.svg",
     "themes":{
         "akeneo-concepts":{
             "title": "Discover Akeneo concepts",
             "img": "illus-concepts.svg",
-            "description": "If you are new to Akeneo, you will want to start here. It's just a few basics to get you started. Remember every Jedi knight was a young Padawan first."
+            "description": "New to Akeneo? Start here with the basics. Remember every Jedi Knight began as a young Padawan!"
         },
         "catalog-settings":{
-            "title": "Build up your catalog",
+            "title": "Building your catalog",
             "img": "illus-catalog.svg",
-            "description": "Having a catalog modelling that is totally tailored to your end-users needs, is really crucial. Before beginning to play with Akeneo, be sure to understand how these few key settings work."
+            "description": "Custom tailoring the catalog model to the needs of your users is absolutely crucial. Before beginning to play with Akeneo, be sure to check out these few key settings."
```

Comment:
Could you find another word for "custom", we avoid it in the PIM native documentation ;). Stg like "sur mesure"